tests/utimensat/01.t

```diff
@@ -0,0 +1,54 @@
+#! /bin/sh
+# $FreeBSD$
+
+desc="utimensat with UTIME_NOW will set the will set typestamps to now"
+
+dir=`dirname $0`
+. ${dir}/../misc.sh
+echo "1..7"
+
+require utimensat
```

Comment:
These calls should be moved up before the test plan summaries -- the reason why is that if the require fails, then it'll print out something like:

<pre>
1..7
1..1
</pre>

This will confuse the TAP parsers. I know this will cause kyua to choke at least (and I think prove will reject the test because of the test plan mismatch).